curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/declare-string-variables.english.md

```diff
@@ -38,11 +38,7 @@ tests:
 <div id='js-seed'>
 
 ```js
-// Example
-var firstName = "Alan";
-var lastName = "Turing";
```

Comment:
Similar to the last comment, should the removed example be placed in the challenge description?

Reply:
I did this in another PR.